packages/inter-protocol/src/interest.js

```diff
@@ -1,11 +1,11 @@
-import { AmountMath } from '@agoric/ertp';
-import { natSafeMath } from '@agoric/zoe/src/contractSupport/index.js';
+import { Fail } from '@agoric/assert';
```

Comment:
this si the deprecated way. You don't need an import to get `assert`, just move this below imports.

```suggestion
const { Fail } = assert;
```

Reply:
the jury's still out on that: https://github.com/Agoric/agoric-sdk/issues/5672